src/fs-helpers.ts

```diff
@@ -99,24 +136,52 @@ function projectInternal(cwd: string) {
     function dir(path: string, cb?: (dir: DirectoryApi) => void) {
       return createDirectory(Path.join(dirPath, path), cb);
     }
+    function readFrom(realFsDirPath: string, targetPath?: string, ignoredPaths?: string[]): void {
+      const targetDir = targetPath ? dir(targetPath) : _dir;
+      readFromFsIntoFixture(realFsDirPath, targetDir);
+    }
+    function getFile(path: string): BaseFile | undefined {
```

Comment:
I'd have expected to avoid any use of `BaseFile` in the public API, would it be better to use `File` here?